packages/core/admin/admin/src/components/MainNav/tests/NavUser.test.tsx

```diff
@@ -1,4 +1,4 @@
-import { screen, render } from '@tests/utils';
+import { screen, render, fireEvent } from '@tests/utils';
```

Comment:
If you can we prefer using the user event instead of the fireEvent in our tests, can you modify the unit test please? Thanks a lot